src/aics-image-viewer/components/App/index.tsx

```diff
@@ -1159,6 +1167,48 @@ export default class App extends React.Component<AppProps, AppState> {
     return { x: 0, y: 0, z: 0 };
   }
 
+  private metadataSelectors: MetadataSelectors = {
+    dimensions: (image) => {
+      const { x, y, z } = image;
+      return { Dimensions: { x: x + "px", y: y + "px", z: z + "px" } };
+    },
+    originalDimensions: (image) => {
+      const { width, height, tiles } = image.imageInfo;
+      return { "Original dimensions": { x: width + "px", y: height + "px", z: tiles + "px" } };
+    },
+    physicalDimensions: (image) => {
+      const { physicalScale, physicalUnitSymbol, normalizedPhysicalSize } = image;
+      const [x, y, z] = normalizedPhysicalSize.toArray().map((dim: number) => dim * physicalScale + physicalUnitSymbol);
+      return { "Physical size": { x, y, z } };
+    },
+    pixelPhysicalSize: (image) => {
+      const { pixel_size, physicalUnitSymbol } = image;
+      const [x, y, z] = pixel_size.map((dim) => dim + physicalUnitSymbol);
+      return { "Physical size per pixel": { x, y, z } };
```

Comment:
is this displayed as 3 rows labeled "x", "y", and "z" respectively?

Reply:
also it should be noted that this is "physical size per original pixel" unless you do some more math.. I think.. worth double checking